### DISCUSSION TITLE
Change model units to accept mm instead of m

### DISCUSSION THREAD

**Discussion by Dervis-vL** [Dlubal-Software/RFEM_Python_Client#261] [Q&A] [answered]
Hi, 

I've looking around in the documentation, asking here is possibly faster :-)

I want to pass my model dimensions in mm instead of m. How do I change this setting?

**Comment by OndraMichal** [comment, marked as answer]
Hi @Dervis-vL,
sadly this is not possible. Using WS we interact directly with RFEM objects wich are using SI dimension. You can test it yourself by changing the Units and decimal Places. WS will still return SI dimensions. But you can create converter that will facilite this.

**Reply by Dervis-vL** [reply]
Both mm and m are SI dimensions. Testing by changing the Units and decimal places is exactly what I want to achieve :-) How do I do this?

**Reply by OndraMichal** [reply]
Wrong. Only meters are SI. See [Wiki](https://en.wikipedia.org/wiki/SI_base_unit) or [NIST](https://www.nist.gov/pml/owm/metric-si/si-units).
You can check it by using default units (m), printing out arbitrary object with coordinates/length and then changing units of length in RFEM to mm and printing it again. You will see the same number. That imply that you can't use mm as input. But it is trivial problem.  When setting the value, just divide the numbers (in mm) by 1000.
